presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -198,4 +205,16 @@ public void run()
             }
         };
     }
+
+    private static boolean shouldAddStackTrace(Exception e)
+    {
+        boolean shouldAddStackTrace = true;
+        if (e instanceof PrestoException) {
+            ErrorCode errorCode = ((PrestoException) e).getErrorCode();
+            if (errorCode.equals(WORKER_RESTARTED) || errorCode.equals(PAGE_TRANSPORT_TIMEOUT) || errorCode.equals(TOO_MANY_REQUESTS_FAILED)) {
```

Comment:
Actually, add a commit before this which adds `equals()` and `hashCode()` to `ErrorCode`, then this can be `EXPECTED_ERRORS.contains(errorCode)`